.github/workflows/rgdc.yml

```diff
@@ -0,0 +1,35 @@
+name: rgdc python client
+on:
+  push:
+    branches: "*"
+    tags: "v*"
+  pull_request:
+    branches: "**"
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel twine
+      - name: Install rgdc
+        working-directory: ./rgdc
+        run: pip install -e .
+      - name: Test (not implemented)
+        working-directory: ./rgdc
+        run: python -c "from rgdc import Rgdc; print('Import went okay!')"
+      - name: Build and Publish to PyPI
+        if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
+        working-directory: ./rgdc
+        env:
+          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+        run: |
+          python setup.py sdist bdist_wheel
+          twine upload --skip-existing dist/*
```

Comment:
What do we need the `bdist_wheel` step for? Since it's just a client library, I think just `sdist` is fine, but I don't feel too strongly about it.

Reply:
We don't need it... but it makes installation faster/easier. Otherwise, when you `pip install rgdc` it would pull the full tarball and build the wheel locally (running `setup.py bdist_wheel` on the client machine). Since this is a pure Python package, the wheel we generate is compatible anywhere and only speeds up the install.

In my experience, I've found it best to include both source and a wheel